src/containers/guest-home-page/how-it-works/HowItWorks.js

```diff
@@ -0,0 +1,51 @@
+import { Box } from '@mui/system'
+import { Typography } from '@mui/material'
+import { useTranslation } from 'react-i18next'
+import { useState } from 'react'
+import { routes } from '~/constants/routes'
+import Stack from '@mui/material/Stack'
+import Switch from '@mui/material/Switch'
+import CardWithButton from '~/components/card-with-button/cardWithButton'
+import { mentorCardBoxArray } from '~/containers/guest-home-page/how-it-works/mentorCardBoxArray'
+import { studentCardBoxArray } from '~/containers/guest-home-page/how-it-works/studentCardBoxArray'
+import { style } from '~/containers/guest-home-page/how-it-works/how-it-works.style'
+
+const sectionId = routes.guestNavBar.howItWorks.label
+
+const HowItWorks = () => {
+  const { t } = useTranslation()
+
+  const [isStudent, setIsStudent] = useState(false)
+
+  const handleChange = () => {
+    setIsStudent(!isStudent)
+  }
+
+  const cardMap = isStudent ? <CardWithButton array={ mentorCardBoxArray } btnText={ 'Start Learning Today' } role={ 'mentor' }  /> : <CardWithButton array={ studentCardBoxArray } btnText={ 'Become a mentor' } role={ 'student' } />

```

Comment:
very long line
create variables and use them in the ternary operator
or transfer it to 3 lines

Reply:
fixed